components/molecules/tabs/components/tab.js

```diff
@@ -49,21 +49,29 @@ const StyledTab = styled.li`
     !active &&
     css`
       cursor: pointer;
-      &:hover {
-        background-color: ${theme.TABS.hoverColour};
-      }
     `}
 
   ${({ disabled, theme }) =>
     disabled &&
     css`
       background-color: ${theme.TABS.disabledColour};
       cursor: no-drop;
-      &:hover {
-        background-color: ${theme.TABS.disabledColour};
-      }
     `}
 
+  &:hover {
```

Comment:
@majid-amiri This refcator looks a little strange - added more conditionals over the inline - was it because the linter was complaining??

Reply:
Exactly - It complains about having two or more `:hover` in one class.